src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -180,6 +181,8 @@ public String convert(Object o) {
 				ItemMeta m = is.getItemMeta();
 				return m.hasDisplayName() ? m.getDisplayName() : null;
 			}
+		} else if (o instanceof World) {
+			return ((World) o).getName();
```

Comment:
```suggestion
			return mark == 1 ? ((World) o).getName() : null;
```
`chat name of player's world` is not intended.

Reply:
I think this will confuse users if we just return nothing

Ex: *why isn't `display name of {_world}` working*

It might be better to have an error during init - opinions welcomed :)

Reply:
Although I don't think majority would use other than `just "name"` at their first try, I agree.

Reply:
> I think this will confuse users if we just return nothing
> 
> Ex: _why isn't `display name of {_world}` working_
> 
> It might be better to have an error during init - opinions welcomed :)

As I said 
> We can't type check at parse time, multiple expression types will result in object.

The syntaxes would have to be split up again like they were 2 years ago before the rewrite.

Reply:
It should be possible to check the type in some cases as long as it isn't a variable IIRC